openexr-rs/src/version.rs

```diff
@@ -0,0 +1,309 @@
+use openexr_sys as sys;
+use std::ops::{BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign};
+use std::os::raw::{c_char, c_int};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct Flag(c_int);
```

Comment:
Would it be easier to just add a dependency on bitflags for this? It's one of those crates like serde that's pretty much standard now